docs/reference/portal.dataProduct.json

```diff
@@ -1427,9 +1496,11 @@
                 "databricksWorkspaceId": "[if(empty(steps('generalSettings').machineLearningDeploymentSettings.databricksId), '', steps('generalSettings').machineLearningDeploymentSettings.databricksId)]",
                 "databricksWorkspaceUrl": "[if(empty(steps('generalSettings').machineLearningDeploymentSettings.databricksWorkspaceUrl.properties.workspaceUrl), '', steps('generalSettings').machineLearningDeploymentSettings.databricksWorkspaceUrl.properties.workspaceUrl)]",
                 "databricksAccessToken": "[if(empty(steps('generalSettings').machineLearningDeploymentSettings.databricksAccessToken), '', steps('generalSettings').machineLearningDeploymentSettings.databricksAccessToken)]",
-                "enableRoleAssignments": "[steps('generalSettings').generalSettings.enableRoleAssignments]",
+                "enableRoleAssignments": "[equals(steps('generalSettings').generalSettings.enableRoleAssignments, 'yes')]",
                 "cognitiveServiceKinds": "[if(empty(steps('generalSettings').generalSettings.cognitiveServiceKinds), parse('[]'), steps('generalSettings').generalSettings.cognitiveServiceKinds)]",
                 "enableSearch": "[steps('generalSettings').generalSettings.enableSearch]",
```

Comment:
```suggestion
                "enableSearch": "[equals(steps('generalSettings').generalSettings.enableSearch, 'yes')]",
```

Reply:
This change was merged into this branch, but you removed it again. See here: https://github.com/Azure/data-product-analytics/pull/121/files

Reply:
Included that change now